locale/action_names.rb

```diff
@@ -1,60 +1,65 @@
 # Autogenerated!
+_("Sync capsule")
```

Comment:
Reordering this is annoying and fixed in https://github.com/theforeman/foreman-tasks/commit/0a8e3fb3fe35ba469c862b5651522319ce6644fd

Reply:
Is this ok to merge? I can make an issue to include fix it here too.

Reply:
It looks like that since I made this version 7.2.0 has been released with the fix. That means it'll be sorted once regenerated, which I just did.